Java Migration/TFS_2018RTW.md

```diff
@@ -0,0 +1,41 @@
+
+
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for TFS 2018 RTW.
+
+![Java Migration flow](flow1.png)
+
+## Step 1: Pause Search indexing
+Go to https://github.com/Microsoft/Code-Search and find the right folder based on the TFS version you are using. For TFS 2018 RTW, go to https://github.com/Microsoft/Code-Search/tree/master/TFS_2018RTW. To pause all indexing, execute the script PauseSearchIndexing.ps1 from Windows PowerShell with administrative privileges. You will be prompted to enter:
+
+* The SQL server instance name where the TFS configuration database resides.
+* The name of the TFS configuration database.
+
+## Step 2: Stop Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-2.4.1\bin"
```

Comment:
Can you check once if the path is correct? I think it should be 16.0 not 15.0 for TFS 2018.

Reply:
fixed it